packages/core/src/sync/sync-api.ts

```diff
@@ -365,6 +365,19 @@ export class SyncApi implements ISyncApi {
     })
   }
 
+  /**
+   * Stop models from being included in the continuous sync
+   * TODO (CDB-2303): Remove existing history sync jobs as well
```

Comment:
could we log a warning that the sync will keep running to completion, and mention that this is a temporary state, and link a TODO to delete the log message when this is done?